### DISCUSSION TITLE
eslint-plugin-react-jss

### DISCUSSION THREAD

**Discussion by afzalsayed96** [cssinjs/jss#1558] [Show and tell]
Hey 👋 

We've created an `eslint` plugin to easily identify and fix the `Warning: [JSS] <Hook />'s styles function doesn't rely on the "theme" argument. We recommend declaring styles as an object instead.` warnings.

Sharing here for anyone that might find it useful:

👉 [GitHub](https://github.com/dashxhq/eslint-plugin-react-jss)
👉 [npm](https://www.npmjs.com/package/eslint-plugin-react-jss)

**Comment by kof** [comment]
LGTM, feel free to add it to this file with a separate category "Linters" https://github.com/cssinjs/jss/blob/master/docs/projects.md
